### ISSUE TITLE
No sound

### ISSUE THREAD

**Issue by marrem** [philippe44/LMS-ShairTunes2W#47]
LMS

```
    Versie: 8.4.0 - 1699423936 @ Wed 08 Nov 2023 07:27:21 AM CET
    Hostnaam: mediaserver
    IP-adres van server: 192.168.10.16
    Besturingssysteem: Debian - NL - utf8
    Platformarchitectuur: armv7l-linux
    Perl versie: 5.36.0 - arm-linux-gnueabihf-thread-multi-64int
    Audio::Scan: 1.06
    IO::Socket::SSL: 2.081
    Database versie: DBD::SQLite 1.58 (sqlite 3.22.0)


    ShairTunes2 (fork) 1.3.3

	Helper binary: shairport_helper_linux-arm
	Encoding parameters: flac, level 3

```

Clients

 Keuken

```
    Model: SqueezeLite
    Type: squeezelite
    Firmware: v1.9.9-1392-pCP
    IP: 192.168.10.169
    MAC Adres: b8:27:eb:92:66:ac
```

Woonkamer_a

```
    Model: SqueezeLite
    Type: squeezelite
    Firmware: v1.9.8-1317
    IP: 192.168.10.14
    MAC Adres: b8:27:eb:6c:bb:d8
```

Source

iPhone 14 pro / IOS 17.1.1

Using either of the above players results in application showing it is playing audio,
but no audio is being heard on the player.

All other functionality of LMS (BBC Sounds plugin, Radio, local files) and players working without any
issues.

[server.log](https://github.com/philippe44/LMS-ShairTunes2W/files/13301306/server.log)
[shairtunes2-b8-27-eb-92-66-ac-flac.log](https://github.com/philippe44/LMS-ShairTunes2W/files/13301307/shairtunes2-b8-27-eb-92-66-ac-flac.log)
[shairtunes2-b8-27-eb-92-66-ac-mp3.log](https://github.com/philippe44/LMS-ShairTunes2W/files/13301315/shairtunes2-b8-27-eb-92-66-ac-mp3.log)
[shairtunes2-b8-27-eb-6c-bb-d8.log](https://github.com/philippe44/LMS-ShairTunes2W/files/13301322/shairtunes2-b8-27-eb-6c-bb-d8.log)

Tried:

- changed buffer / timings in plugin settings
- checked ping times between LMS - squeezelite players
   - connected by wire:
 ```
PING mpw.homenet (192.168.10.14) 56(84) bytes of data.
64 bytes from mpw.homenet (192.168.10.14): icmp_seq=1 ttl=64 time=0.614 ms
64 bytes from mpw.homenet (192.168.10.14): icmp_seq=2 ttl=64 time=0.585 ms
64 bytes from mpw.homenet (192.168.10.14): icmp_seq=3 ttl=64 time=0.537 ms
64 bytes from mpw.homenet (192.168.10.14): icmp_seq=4 ttl=64 time=0.529 ms
^C
--- mpw.homenet ping statistics ---
4 packets transmitted, 4 received, 0% packet loss, time 3004ms
```

   - connected by wifi: 
```
marc@mediaserver:/var/log/squeezeboxserver $ ping 192.168.10.169
PING 192.168.10.169 (192.168.10.169) 56(84) bytes of data.
64 bytes from 192.168.10.169: icmp_seq=1 ttl=64 time=6.93 ms
64 bytes from 192.168.10.169: icmp_seq=2 ttl=64 time=4.91 ms
64 bytes from 192.168.10.169: icmp_seq=3 ttl=64 time=6.63 ms
```

**Comment by philippe44** [comment]
Try removing "HTTP fill option"

**Comment by marrem** [comment]
Yes, that solved the problem. Now both players (wired and wifi) work again without issues. I'm using the iPhone14 as before.

I has worked for years, that setup. Only a month ago when I updated the shairtunes plugin, it stopped working. Yesterday I decided to start with a fresh RaspberryPi Os and LMS. But that didn't solve it at first. But now after deselecting that HTTP fill option everything works fine again. Thanks.
